src/itest/java/uk/gov/companieshouse/company/links/CucumberFeaturesRunnerITest.java

```diff
@@ -8,7 +8,7 @@
 
 @RunWith(Cucumber.class)
 @CucumberOptions(
-        features = "src/itest/resources/features",
+        features = "src/itest/resources/features/company-links-charges-error-retry.feature",
```

Comment:
Curios if we need to mention all feature files explicitly instead of just folder?